src/Evaluators/reduced_evaluator.jl

```diff
@@ -82,12 +82,12 @@ function ReducedSpaceEvaluator(
         append!(g_max, cu)
     end
 
-    obj_ad = AdjointStackObjective(model)
+    obj_ad = pullback_objective(model)
     state_ad = FullSpaceJacobian(model, power_balance)
     VT = typeof(g_min)
-    cons_ad = AdjointPolar{VT}[]
+    cons_ad = AutoDiff.PullbackMemory[]
     for cons in constraints
-        push!(cons_ad, AdjointPolar(model))
+        push!(cons_ad, AutoDiff.PullbackMemory(model, cons, VT))
     end
```

Comment:
In AD literature this is usually called stack or tape. But pullback seems to be the new Julia slang. Below where what is now called the stack is defined I made another comment.

Reply:
I have renamed `PullbackMemory` as `TapeMemory`